spec/system/claims/schools/users/remove_a_user_spec.rb

```diff
@@ -1,74 +1,172 @@
 require "rails_helper"
 
 RSpec.describe "Remove a user", service: :claims, type: :system do
-  let!(:school) { create(:claims_school) }
-  let!(:user) do
-    create(:claims_user) { |user| school.users << user }
-  end
-  let!(:current_user) do
-    create(:claims_user) { |user| school.users << user }
-  end
+  scenario "Claims user removes user from a school" do
+    given_a_school_exists_with_claims_users
+    and_i_am_signed_in
+
+    when_i_click_on_users
+    then_i_see_the_claim_users_page
+
+    when_i_click_on_a_user
+    then_i_see_the_details_page_for_the_user
 
-  scenario "I sign in as a user and remove a user from a school" do
-    given_i_am_signed_in_as(current_user)
-    and_i_navigate_to_user_details_page(user)
     when_i_click_on_remove_user
-    then_i_am_taken_to_a_removal_confirmation_page(school, user)
+    then_i_see_the_removal_confirmation_page
+
     when_i_click_on_remove_user
-    then_the_user_has_been_removed_from_the_school(user)
+    then_i_see_the_claim_users_page_with_the_user_removed
   end
 
-  scenario "I sign in as a user and I do not see the link to remove myself from a school" do
-    given_i_am_signed_in_as(current_user)
-    and_i_navigate_to_user_details_page(current_user)
-    then_i_do_not_see_the_remove_user_button
+  scenario "Claims user cannot see remove user link for their own user" do
+    given_a_school_exists_with_claims_users
+    and_i_am_signed_in
+
+    when_i_click_on_users
+    then_i_see_the_claim_users_page
+
+    when_i_click_on_my_own_name
+    then_i_see_my_own_details_page_with_no_remove_user_link
   end
 
-  scenario "I sign in as a user and I cannot remove myself from a school" do
-    given_i_am_signed_in_as(current_user)
-    and_i_navigate_to_user_removal_page(school, current_user)
-    then_i_am_forbidden_to_perform_this_action
+  scenario "Claims user prevented from accessing the remove user route for their own user" do
+    given_a_school_exists_with_claims_users
+    and_i_am_signed_in
+
+    and_i_navigate_to_my_own_user_removal_page
+    then_i_am_alerted_that_i_cannot_perform_this_action
   end
 
   private
 
-  def given_i_am_signed_in_as(user)
-    sign_in_as user
+  def given_a_school_exists_with_claims_users
+    @user_anne = create(:claims_user, first_name: "Anne", last_name: "Wilson", email: "anne_wilson@education.gov.uk")
+    @user_barry = create(:claims_user, first_name: "Barry", last_name: "Garlow", email: "barry_garlow@education.gov.uk")
+
+    @shelbyville_school = create(
+      :claims_school,
+      name: "Shelbyville Elementary",
+      users: [@user_anne, @user_barry],
+    )
+  end
+
+  def and_i_am_signed_in
+    sign_in_as(@user_anne)
   end
 
-  def and_i_navigate_to_user_details_page(user)
+  def when_i_click_on_users
```

Comment:
```suggestion
  def when_i_navigate_to_users
```
Use `navigate` to describe when you are using a navigation.